se/sics/mspsim/platform/tyndall/TyndallNode.java

```diff
@@ -153,7 +153,19 @@ public void setupNode() {
                 SerialMon serial = new SerialMon((USARTSource)usart, "USCI A0 Port Output");
                 registry.registerComponent("serialgui", serial);
             }
-        }
+        } else {
+            // Send serial output to stdout
+            IOUnit usart = cpu.getIOUnit("USCI A0");
+            if (usart instanceof USARTSource) {
```

Comment:
This can become `if (usart instanceof USARTSource usartSource) {` in the Cooja repository, and the line with the cast below omitted.